packages/commercetools/api-client/src/types/Api.ts

```diff
@@ -122,6 +125,7 @@ interface ApiMethods {
   updateCartQuantity ({ id, version }: CartDetails, product: LineItem): Promise<CartResponse>;
   updateShippingDetails (cart: Cart, shippingDetails: Address): Promise<CartResponse>;
   isGuest: () => boolean;
+  getStores(params: CustomQuery): Promise<StoreQueryResult>;
```

Comment:
Maybe we should nest it in case we want to add additional params in the future?
```ts
  getStores(params: { customQuery: CustomQuery }): Promise<StoreQueryResult>;
```